packages/yoastseo/src/stringProcessing/relevantWords.js

```diff
@@ -224,6 +246,7 @@ export default {
 	getRelevantWords: getRelevantWords,
 	getRelevantWordsFromPaperAttributes: getRelevantWordsFromPaperAttributes,
 	getRelevantCombinations: getRelevantCombinations,
+	getRelevantCombinationsForInsights: getRelevantCombinationsForInsights,
```

Comment:
I think this can be simplified as thus:
```suggestion
	getRelevantCombinationsForInsights,
```